docker/package/model.py

```diff
@@ -584,9 +585,16 @@ def gen_makefile(self, out):
         file_contents = """
 .PHONY: install
 
+BINDIR=/usr/bin
+
 tezos-baking:
 
-install: tezos-baking
+tezos-baking-wizard:
+	chmod +x $(DESTDIR)$(BINDIR)/tezos-baking-wizard
+
+install: tezos-baking tezos-baking-wizard
+	mkdir -p $(DESTDIR)$(BINDIR)
+	cp $(CURDIR)/tezos_baking_wizard.py $(DESTDIR)$(BINDIR)/tezos-baking-wizard
```

Comment:
To complete my comment above
```suggestion
	cp $(CURDIR)/tezos-baking-wizard $(DESTDIR)$(BINDIR)/tezos-baking-wizard
```